libcst/codemod/commands/rename.py

```diff
@@ -0,0 +1,173 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# pyre-strict
+import argparse
+from typing import Optional, Sequence, Union
+
+import libcst as cst
+from libcst.codemod import CodemodContext, VisitorBasedCodemodCommand
+from libcst.codemod.visitors import AddImportsVisitor, RemoveImportsVisitor
+from libcst.helpers import get_full_name_for_node
+from libcst.metadata import QualifiedName, QualifiedNameProvider, QualifiedNameSource
+
+
+class RenameCommand(VisitorBasedCodemodCommand):
+    """
+    Rename all instances of a variable or import.
+    """
+
+    DESCRIPTION: str = (
+        "Rename all instances of a variable or import. "
+        + "The existing import may not be removed and RenameCommand needs more work to be perfect."
```

Comment:
The `not be removed` issue should be fixed by this PR. Let's update the description.